tests/integration/commands/export/rst/08_custom_field/expected/input.rst

```diff
@@ -0,0 +1,16 @@
+.. _A-1:
+
+.. list-table::
+    :align: left
+    :header-rows: 0
+
+    * - **UID:**
+      - A-1
+
+    * - **META_TEST:**
+      - Yes
+
```

Comment:
I know, it can be quite tricky to get right, but could you ensure that the generator always generates no more than 1 space between logical blocks? You might need to play around with jinja's `{%-` compared to `{%`. Try to follow how I am doing it in the rest of the `strictdoc/export/rst/templates/requirement.jinja.rst`.

Reply:
yeah... it is tricky. let me see if I can wrestle it

Reply:
Please do. If it gets hard, just drop it and let me know, and I could do another round myself. You might have noticed that I had to introduce the `{%- if true` hack to create a non-eatable string.

Reply:
> Please do. If it gets hard, just drop it and let me know, and I could do another round myself. You might have noticed that I had to introduce the `{%- if true` hack to create a non-eatable empty line.

so... I tried (probably more than reasonable) to get the extra line after multiple single-line meta fields to go away. Everything I tried either a) got rid of that extra newline _and_ the single new line after a single sngle-line meta (e.g. UID only) or b) kept the single newline in the prev case but added the extra newline in multi single line meta tags...

Reply:
> with respect to the checks failing: those are all whitespace differences in the rst exports right? Can we change the integration test commands to e.g. `diff -burp` instead?

I liked your idea of a `long-bearing comment`. Unfortunately, it doesn't remove the trailing whitespace compared to what my hack does. The rst integration tests are passing now as is.